examples/simple_example/main.tf

```diff
@@ -69,7 +69,9 @@ module "alloy-db" {
     instance_id = "primary-instance-1",
     client_connection_config = {
       require_connectors = false
-      ssl_config         = "ALLOW_UNENCRYPTED_AND_ENCRYPTED"
+      ssl_config = {
+        ssl_mode = "ALLOW_UNENCRYPTED_AND_ENCRYPTED"
+      }
```

Comment:
Shouldn't it be just 
```hcl
ssl_mode = `"ALLOW_UNENCRYPTED_AND_ENCRYPTED"`
```
?

Source seems not using the `ssl_config` attribute
```hcl
  dynamic "client_connection_config" {
    for_each = lookup(var.primary_instance, "ssl_mode", null) != null || lookup(var.primary_instance, "require_connectors", null) != null ? ["client_connection_config"] : []

    content {
      require_connectors = try(var.primary_instance.require_connectors, null)
      ssl_config {
        ssl_mode = try(var.primary_instance.ssl_mode, null)
      }
    }
  }
```

Reply:
But in the actual module `ssl_config` is not used 
* https://github.com/GoogleCloudPlatform/terraform-google-alloy-db/blob/68e3aeac524d9eebeba17e58c0e451d6b9351dc0/main.tf#L158-L167
* https://github.com/GoogleCloudPlatform/terraform-google-alloy-db/blob/68e3aeac524d9eebeba17e58c0e451d6b9351dc0/main.tf#L195-L203

Reply:
@rajc242 it should be 
```
ssl_mode = `"ALLOW_UNENCRYPTED_AND_ENCRYPTED"`
```

Reply:
Fix applied.